### PR TITLE
BS - 23

### PR DESCRIPTION
**Pull request by matthewleemle**

### CODE REVIEW

File: controllers/README.md [hunk 1]
```diff
@@ -222,6 +222,27 @@ HTTP Response:
 }
 ```
 
+
+### Getting the responding to list of a userRespondingTo
+
+GET "/users/{id}/responding_to"
```
Comment (mxnyee) [inline controllers/README.md]:
```suggestion
GET "/users/{id}/responding-to"
```

File: controllers/README.md [hunk 2]
```diff
@@ -222,6 +222,27 @@ HTTP Response:
...
+
+HTTP Response:
+
+```
+{
+    "Responding_To": [
```
Comment (mxnyee) [inline controllers/README.md]:
```suggestion
    "respondingTo": [
```

File: controllers/README.md [hunk 3]
```diff
@@ -222,6 +222,27 @@ HTTP Response:
 }
 ```
 
+
+### Getting the responding to list of a userRespondingTo
```
Comment (mxnyee) [inline controllers/README.md]:
```suggestion
### Getting the users a responder is watching for
```

File: controllers/README.md [hunk 4]
```diff
@@ -222,6 +222,27 @@ HTTP Response:
...
+
+HTTP Response:
+
+```
+{
+    "Responding_To": [
+        {
+            "username": string,
+            "id": string,
+            "naloxoneAvailability": boolean
```
Comment (mxnyee) [inline controllers/README.md]: We don't need the availability field in the response

Comment (matthewleemle) [reply controllers/README.md]: @Asiapenguin Confirming we don't need the availability field in the response?

Comment (Asiapenguin) [reply controllers/README.md]: We dont need it. The screens dont have availability status.

Comment (Asiapenguin) [reply controllers/README.md]: Just an aside, thats not the right attribute for availability status anyway lol.

Comment (matthewleemle) [reply controllers/README.md]: got it lol

**Pull request merged**